scripts/sauce/sauce_connect_setup.sh

```diff
@@ -21,6 +20,11 @@ CONNECT_STDOUT="$LOGS_DIR/sauce-connect.stdout"
 CONNECT_STDERR="$LOGS_DIR/sauce-connect.stderr"
 
 # Get Connect and start it
+if [ `uname -s` = "Darwin" ]; then # If user is running Mac, download Mac version
```

Comment:
I had never seen "Darwin" in this context before. Can we include a link?
https://en.wikipedia.org/wiki/Darwin_(operating_system)